mindmeld/core.py

```diff
@@ -406,7 +439,9 @@ class ProcessedQuery:
         confidence (dict): A dictionary of the class probas for the domain and intent classifier
```

Comment:
TIL: We keep track of whether a query is human labelled or predicted! Do we use `is_gold` anywhere?